packages/cfd/src/Containers/__tests__/cfd-financial-stp-real-account-signup.spec.js

```diff
@@ -33,7 +33,6 @@ jest.mock('../../Components/cfd-poi', () =>
         </div>
     ))
 );
-
```

Comment:
@yauheni-deriv we might not need the change in this file if it's only about removing 1 extra line :) But it's up to you )